tests/slice_plotter_presenter_test.py

```diff
@@ -17,34 +17,35 @@ def setUp(self):
         # Setting up view for slice plotter window
         self.slice_plotter = mock.create_autospec(SlicePlotter)
         self.slice_view = mock.create_autospec(SlicePlotterView)
+        self.plotting_module = mock.Mock(spec=["imshow","xlabel","ylabel"])
```

Comment:
I think the module can be our interface for now. It allows us to mock out the functions so should be sufficient for testing